modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeFtlGetPortletGroupIdMethodCheck.testftl

```diff
@@ -0,0 +1,12 @@
+<#-- FreeMarker Template -->
+
+<#assign
+	group = themeDisplay.getScopeGroupId()
+/>
+<@liferay_aui.col width=25>
```

Comment:
@TamyrisTorres, can you please remove this taglib from this test?

It's not so necessary considering the test's scope

Thanks